cernopendata/templates/cernopendata_records_ui/records/record_detail.html

```diff
@@ -101,13 +101,12 @@ <h2>Related Datasets</h2>
     {% if record.distribution and (record.distribution.number_events or record.distribution.number_files or record.distribution.size) %}
         <div class="row">
             <div class="col-md-12">
-                <h2> Characteristics </h2>
-                <label> Dataset: </label>
+                <h2>Dataset Characteristics </h2>
                 {% if record.distribution.number_events %}
-                <span><strong>{{record.distribution.number_events}}</strong> events</span>
+                <span><strong>{{record.distribution.number_events}}</strong> events</span><br>
                 {% endif %}
                 {% if record.distribution.number_files %}
-                <span><strong>{{record.distribution.number_files}}</strong> files</span>
+                <span><strong>{{record.distribution.number_files}}</strong> files</span><br>
```

Comment:
The line break here makes the page longer, and since we have enough horizontal space, let's rather use it...

Reply:
+1